src/L.Control.Heightgraph.js

```diff
@@ -907,13 +932,58 @@ L.Control.Heightgraph = L.Control.extend({
                 this[param].style('display', 'none');
             }
     },
+    /*
+     * Handles the mouseover the map and displays distance and altitude level.
+     * Since this does a lookup of the point on the graph
+     * the closest to the given latlng on the provided event, it could be slow.
+     */
+    _mapMousemoveHandler(evt) {
+        if (this._areasFlattended == false) {
+            return;
+        }
+
+        // initialize the vars for the closest item calculation
+        let closestItem = null;
+        // large enough to be trumped by any point on the chart
+        let closestDistance = 2 * Math.pow(100, 2);
+
+        for (i = 0; i < this._areasFlattended.length; i++) {
+            let item = this._areasFlattended[i];
+
+            let latDiff = evt.latlng.lat - item.latlng.lat;
+            let lngDiff = evt.latlng.lng - item.latlng.lng;
+
+            // first check for an exact match; it's simple and avoid further calculations
+            if (latDiff == 0 && lngDiff == 0) {
```

Comment:
Maybe we better check for diff <= .000001 or something like this? Exact matches with floating points numbers are guess work. In this case we might need to Math.abs(diff) before?

Reply:
done; let me know if it's better now